conf/application.conf

```diff
@@ -175,6 +177,7 @@ urls {
       certOfResidence = "https://www.gov.uk/guidance/get-a-certificate-of-residence"
       closeCompany = "https://www.gov.uk/closing-a-limited-company"
       makeCompanyDormant = "https://www.gov.uk/dormant-company/dormant-for-corporation-tax"
+      covidGuidance = "https://www.gov.uk/government/publications/guidance-to-employers-and-businesses-about-covid-19"
```

Comment:
This is not currently being used. Do we want to use it ore remove it?

Reply:
Now used in covid_links partial